pywbemtools/pywbemcli/_cmd_statistics.py

```diff
@@ -0,0 +1,338 @@
+
+    This may fail if the server does not manage statistics or does not allow
+    a client to modify the state of statistics gathering.
+
+    This does not impact pywbem local client statistics gathering. See the
+    --timestats general option.
+    """
+    # pylint: disable=too-many-function-args
+
+    context.execute_cmd(lambda: cmd_statistics_server_off(context))
+
+
+@statistics_group.command('server-stats-show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def server_stats_show(context):
+    """
+    Display statistics gathered by server.
+
+    Display the current statistics gathered by the server if statistics
+    gathering is implemented for the current WBEM server.
+
+    Presents a table of the gathered statistical information.
+
+    This is not implemented. See issue #895
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_server_stats_show(context))
+
+
+@statistics_group.command('show', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_show(context):
+    """
+    Display statistics gathered by client.
+
+    Display the current statistics including local client gathered statistics
+    and server operation times (if server statistics are enabled).
+
+    Statististics are always displayed as a table independent of output-format
+    defined.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_show(context))
+
+
+@statistics_group.command('status', cls=PywbemcliCommand,
+                          options_metavar=CMD_OPTS_TXT)
+@add_options(help_option)
+@click.pass_obj
+def statistics_status(context):
+    """
+    Show statistics enabled status for client and server.
+
+    Show enabled status on statistics gathering for this client and the
+    current server.
+    """
+    # pylint: disable=too-many-function-args
+    context.execute_cmd(lambda: cmd_statistics_status(context))
+
+
+###############################################################
+#         Server cmds  Common components
+###############################################################
+
+OBJMGR_CLN = "CIM_ObjectManager"
+OBJMGR_STAT_PROPERTY_NAME = "GatherStatisticalData"
+
+
+def test_conn_exists(context):
+    """
+    Test if server defined.
+
+    Returns if server defined
+
+    Exception:
+       click.Exception if no current server.
+    """
+    if not context.conn:
+        raise click.ClickException(
+            'No server defined in current context.')
+
+
+def get_objmgr_inst(context):
+    """
+    Return the state of statistics gathering on the server.
+    """
+    conn = context.conn
+    try:
+        interop_ns = context.wbem_server.interop_ns
+    except Error as er:
+        raise click.ClickException(
+            'Server connection failed. Exception: {}'.format(er))
```

Comment:
Since this could be the first command executed, it could return a wide variety of things outside of just inability to get the interop namespace.  I changed the message to reflect we were trying to find the interop namespace.